knowledge-base/cs0104-error-pdf-format-provider.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Resolving Compiler Ambiguous Reference Error CS0104 with PdfFormatProvider 
+description: Learn how to resolve the ambiguous reference error (Compiler Error CS0104) when using PdfFormatProvider in Telerik UI for WinForms.
+type: troubleshooting
+page_title: Compiler Error CS0104 with PdfFormatProvider in Telerik UI for WinForms
+slug: cs0104-error-pdf-format-provider-winforms
+tags: troubleshooting, PdfFormatProvider, Compiler Error, CS0104,  ambiguous, reference
+res_type: kb
+
+---
+
+## Environment
+
+| Version | Product | Author |
+| --- | --- | ---- |
+| 2023.3.1106 | RadPdfProcessing |[Desislava Yordanova](https://www.telerik.com/blogs/author/desislava-yordanova)|
+
+## Description
+When using the **PdfFormatProvider** in your project, you may encounter one of the following [Compiler Error CS0104](https://learn.microsoft.com/en-us/dotnet/csharp/misc/cs0104?f1url=%3FappId%3Droslyn%26k%3Dk(CS0104)) messages during the application build:
+
+- **'PdfFormatProvider' is an ambiguous reference between 'Telerik.Windows.Documents.Fixed.FormatProviders.Pdf.PdfFormatProvider' and 'Telerik.Windows.Documents.Flow.FormatProviders.Pdf.PdfFormatProvider'**
+
+- **'PdfFormatProvider' is an ambiguous reference between 'Telerik.Windows.Documents.Fixed.FormatProviders.Pdf.PdfFormatProvider' and 'Telerik.Windows.Documents.Spreadsheet.FormatProviders.Pdf.PdfFormatProvider'**
+
+Usually, such an error may occur when the following **using** statements are imported:
+
+```csharp
+using Telerik.Windows.Documents.Fixed.FormatProviders.Pdf;
+using Telerik.Windows.Documents.Flow.FormatProviders.Pdf;
+using Telerik.Windows.Documents.Spreadsheet.FormatProviders.Pdf;
+
+PdfFormatProvider provider = PdfFormatProvider();
+   ```
+## Solution
+
+Telerik Document Processing Libraries offer PdfFormatProvider which makes it easy to import/export a [RadFixedDocument](https://docs.telerik.com/devtools/document-processing/libraries/radpdfprocessing/model/radfixeddocument) from/to PDF format, preserving the entire document structure and formatting. It is available for: 
```

Comment:
The provided information covers the PdfProcessing import-export but when working with Workbook and/or RadFlowDocument we have only export. From time to time, clients are asking to import a PDF file into a RadFlowDocument so we need to be careful to not mislead them.